views/default/admin/users/newest/time_cell.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Render a TD element for an item
+ *
+ * @uses $vars['item']      The item being rendered
+ * @uses $vars['item_vars'] Vars received from the page/components/table view
+ * @uses $vars['attrs']     Array of attributes for the TD
+ */
+
+if (empty($vars['item'])) {
+	// requested as an admin page
+	forward('', '404');
```

Comment:
Also the default cell render has to accept all kinds of item objects. If we gave views like `user/elements/time_created`, I suppose they'd have to bail if not passed item as a user.